src/index.html

```diff
@@ -188,5 +188,14 @@ <h1>Made with love.</h1>
       ga('create', 'UA-67015100-1', 'auto');
       ga('send', 'pageview');
     </script>
+    <a href="https://github.com/get-alex/alex" class="github-corner" aria-label="View source on GitHub">
+      <svg width="80" height="80" viewBox="0 0 250 250" style="fill:#151513; color:#fff; position: absolute; top: 0; border: 0; right: 0;" aria-hidden="true">
```

Comment:
Could you remove this inline style as well? You could select the element from CSS with `.github-corner svg { /* all the inline styles */ }`